lisa/tools/iperf3.py

```diff
@@ -47,59 +53,258 @@ def install(self) -> bool:
             self._install_from_src()
         return self._check_exists()
 
-    def run_as_server(self, daemon: bool = True) -> None:
+    def run_as_server_async(
+        self,
+        port: int = 0,
+        format: str = "",
+        interval_seconds: int = 0,
+        output_json: bool = False,
+        one_off: bool = False,
+        daemon: bool = True,
+    ) -> Process:
         # -s: run iperf3 as server mode
         # -D: run iperf3 as a daemon
         cmd = " -s"
         if daemon:
-            cmd += " -D"
-        self.run(
-            cmd,
-            force_run=True,
-            sudo=True,
-            shell=True,
+            cmd += " -D "
+        if one_off:
+            cmd += " -1 "
+        if output_json:
+            cmd += " -J "
+        if interval_seconds:
+            cmd += f" -i{interval_seconds} "
+        if format:
+            cmd += f" -f {format} "
+        if port:
+            cmd += f" -p {port} "
+        process = self.node.execute_async(
+            f"{self.command} {cmd}", shell=True, sudo=True
+        )
+        return process
+
+    def run_as_server(
+        self,
+        seconds: int = 10,
+        port: int = 0,
+        format: str = "",
+        interval_seconds: int = 0,
```

Comment:
give a recommended default value.

Reply:
there are other cases user iperf3, but don't use this parameter

Reply:
1. It looks the seconds is not used actually.
2. The name seconds and interval_seconds are confusing. It's similar with format and use_json_format.
3. If some parameters are conflict, we can split it to two methods, like run_as_server_by_interval and run_as_server_by_somethingelse.

Reply:
remove seconds